dist/index.html

```diff
@@ -18,10 +18,10 @@
       <p class="spaceship"></p>
     </div>
     <div class="planet item">
-      <p>Planets</p>
+      <a href="#">ABOUT</a>
     </div>
     <div class="race item">
-      <p>Races</p>
+      <a href="#">CONTACT</a>
```

Comment:
Kindly change your classes on these elements to something related to what you have inside the tags, so the code will be more semantic.

Reply:
## Hello @Lordkaito ,
### Thank you for your feedback.
- Changes will be implemented immediately.